packages/appium/docs/en/guides/execute-methods.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Execute Methods
+---
+
+Because the scope of commands implemented in Appium drivers is broader than the scope of commands
+defined by the W3C WebDriver spec, Appium needs a way for these "extended" commands to be accessed
+by client libraries. There are two main strategies for this:
+
+1. Appium drivers define new W3C-compatible API routes, and Appium clients are updated to include
+   support for those new routes.
+2. Appium drivers define so-called "Execute Methods" which provide access to new functionality by
+   overloading the existing `Execute Script` command which is already available in any WebDriver-
```

Comment:
```suggestion
   overloading the existing "Execute Script" command which is already available in any WebDriver-
```

we should be consistent with the punctuation, whatever it is

Reply:
so here's my rule for this:
1. `"Execute Method"` or `Execute Method` -- this is a non-official name that we came up with, hence it's not in backticks
2. `Execute Script` -- this is the official name of the W3C command, hence it's in backticks